server/chart_config.json

```diff
@@ -57,6 +57,18 @@
     "isOverview": true,
     "isChoropleth": true
   },
+  {
+    "category": "Economics",
+    "titleId": "Unemployment rate: urban vs rural",
+    "title": "Unemployment rate: urban vs rural",
+    "description": "Percentage of the population, of working age, who are unemployed in urban vs rural areas.",
```

Comment:
"Percentage of the working age population that is unemployed in urban vs. rural areas" ?